### PR TITLE
Prep for 0.1.0 release

### PR DESCRIPTION
**Pull request by cdettmering**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
-version = 0.0.1a2
+version = 0.1.0
+author = Miguel Alonso Jr
+author_email = drmiguelalonsojr@gmail.com
+maintainer = Chad Dettmering
+maintainer_email = chad.dettmering@gmail.com
 description = A general purpose library for training any type of GPT model. Support for gpt-1, gpt-2, and gpt-3 models.
 long_description = file: docs/README.md
 long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
+keywords = ai, ml, rl, gpt
```
Comment (miguelalonsojr) [inline setup.cfg]: add 'llm' and 'rlhf' to the keywords

File: setup.cfg [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+license = MIT
+classifiers =
```
Comment (miguelalonsojr) [inline setup.cfg:13]: add Development Status :: 4 - Beta to the classifiers. 4 is beta status. we'll be here until we hit a 1.0 release

**Review by miguelalonsojr — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (cdettmering) [reply setup.cfg]: Done

File: setup.cfg (hunk 2 above)
Comment (cdettmering) [reply setup.cfg:13]: Done

**Review by miguelalonsojr — APPROVED** [review]
LGTM

**Pull request merged**